streams/src/main/java/org/apache/kafka/streams/processor/assignment/assignors/StickyTaskAssignor.java

```diff
@@ -40,13 +41,17 @@
 import org.apache.kafka.streams.processor.assignment.TaskAssignmentUtils;
 import org.apache.kafka.streams.processor.assignment.TaskAssignor;
 import org.apache.kafka.streams.processor.assignment.TaskInfo;
+import org.apache.kafka.streams.processor.assignment.TaskTopicPartition;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 
 public class StickyTaskAssignor implements TaskAssignor {
     private static final Logger LOG = LoggerFactory.getLogger(StickyTaskAssignor.class);
 
+    public static final int DEFAULT_STATEFUL_TRAFFIC_COST = 1;
+    public static final int DEFAULT_STATEFUL_NON_OVERLAP_COST = 10;
```

Comment:
why "stateful"? I also think we should use the word "sticky" in there somewhere since we'll be using these right next to where we have similarly named variables for other assignor types. eg `DEFAULT_STICKY_TRAFFIC_COST` or something like that

Reply:
I just copied these from the original internal StickyTaskAssignor, for the sake of consistency. I'll change them to `DEFAULT_STICKY_*` though.

Reply:
so weird. why did they name it "stateful"? both of these assignors, and AFAICT the configs themselves, pertain to both stateless and stateful tasks... 🤷‍♀️